### ISSUE TITLE
Creating new redirector in Cloud plugin hangs application

### ISSUE THREAD

**Issue by khyberspache** [preludeorg/operator-support#282]
Creating new redirector hangs the application sometimes. All buttons become non-functional and the below errors appear:

![image](https://user-images.githubusercontent.com/43142871/126707235-421329cd-c8a6-4b38-8975-da73f842d977.png)

![image](https://user-images.githubusercontent.com/43142871/126707283-33ce423d-cc36-408e-8f40-5f722aa6de2a.png)

**Comment by privateducky** [comment]
I haven't experienced this yet - can you give some repo steps?

**Comment by lzimm** [comment]
yeah i can't repro this either. i wonder if you're somehow getting into a state where the sidebar stack has some hidden stuff in it already somehow?

**Comment by bfuzzy1** [comment]
This is going to be a tough one to reproduce potentially. I've noticed that it is happening at random times on my end.

**Comment by privateducky** [comment]
@lzimm can you try to flush this one out? seems like it comes/goes across different parts of the app at "random". so unlikely a redirector issue but more a load issue around sidebars..?

**Comment by lzimm** [comment]
i've been trying but i haven't been able to reproduce it no matter how hard i try. so until we have some more consistent repro steps i'm not sure what to do :(

**Comment by privateducky** [comment]
Closing. This one hasn't proven to contain repro steps... please re-open if we see it.
